app/controllers/arclight/repositories_controller.rb

```diff
@@ -6,5 +6,12 @@ class RepositoriesController < ApplicationController
     def index
       @repositories = Arclight::Repository.all
     end
+
+    def show
+      repository = Arclight::Repository.find_by!(slug: params[:id])
+      redirect_to search_action_path(
```

Comment:
The way that I read #329 is that `/repositories/:id` should show the same content as the search, not redirect there. @anarchivist is that correct?